utils/cli-chromedriver-manager-update.js

```diff
@@ -0,0 +1,3 @@
+const chromeDriverManager = require('../cli/utils/chromedriver-manager');
```

Comment:
This file allows us to execute the code that was refactored from `cli/e2e.js` to `cli/utils/chromedriver-manager.js`.